src/main/kotlin/org/rust/lang/core/psi/RustElementFactory.kt

```diff
@@ -19,4 +20,14 @@ object RustElementFactory {
         val file = createFileFromText(project, "mod $modName;")
         return file?.childOfType<RustModDeclItem>()
     }
+
+    fun createOuterAttr(project: Project, attrContents: String): RustOuterAttr? {
+        val file = createFileFromText(project, "#[$attrContents] struct Dummy;")
+        return file?.childOfType<RustOuterAttr>()
+    }
+
+    fun createMeta(project: Project, metaContents: String): RustMetaItem? {
```

Comment:
This function is unused. Let's remove it now. We'll reintroduce it when we need this functionality.